ohhamma/react-todo/src/App.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import TodoContainer from './pages/Todo/TodoContainer';
+import TodoHead from './pages/Todo/TodoHead';
+import TodoForm from './pages/Todo/TodoForm';
+import TodoList from './pages/Todo/TodoList';
+import './App.css';
+import './pages/Todo/Todo.css';
+
+function App() {
+  const TODOS_KEY = "todos";
+
+  const saveTodos = () => {
+    localStorage.setItem(TODOS_KEY, JSON.stringify(todos));
+  }
+
+  const loadTodos = () => {
+    const savedTodos = localStorage.getItem(TODOS_KEY);
+    if (savedTodos !== null) {
+      const parsedTodos = JSON.parse(savedTodos);
+      return parsedTodos;
+    }
+    else return [];
+  }
```

Comment:
```suggestion
  const loadTodos = () => {
    const savedTodos = localStorage.getItem(TODOS_KEY);
    return JSON.parse(savedTodos) || [];
  };
```

이렇게 쓰면 더 깔끔할 수 있겠죠!!